### DISCUSSION TITLE
v3.0.1

### DISCUSSION THREAD

**Discussion by Daveiano** [Daveiano/weewx-wdc#128] [Announcements]
- Bugfix: Some built-in fields require adding an entry to [[Icons]] GH-124
- Added icon for no2, pm1_0, pm2_5 and pm10_0
- Bugfix: Unexpected IndexError in get_unit_label GH-123
- Bugfix: Resolved a caching issue for radar images (service worker was caching too much here)

## No changes made to skin.conf since 3.0.0

**Full Changelog**: https://github.com/Daveiano/weewx-wdc/compare/v3.0.0...v3.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Daveiano/weewx-wdc/releases/tag/v3.0.1'>v3.0.1</a>.</em>

**Comment by JonFear** [comment]
Thank you David.

Query, can you recommend a good editor that will allow a novice to write my about and webcam pages more easily than using vim and nano?

I have sent a small token of my appreciation.

Best wishes

Jon

**Reply by Daveiano** [reply]
Thank you very much!

I would recommend [Visual Studio Code](https://code.visualstudio.com/). It's a really powerful tool for beginners and for pros. I am using it myself for a long time now. 

Dave

**Reply by JonFear** [reply]
I have that at work. I will take a look tomorrow...

J

**Comment by JonFear** [comment]
Another thought. Can we have some way of preserving the modified skin and about files? These are overwritten on refresh. When these pages are updated by the user, the work is lost at update. Is there some way of perhaps having a defaults file somewhere that the code looks for? For instance, the radar code needs to be preserved...

Jon

**Reply by Daveiano** [reply]
You can persist changes to the skin.conf in the weewx.conf, see here: https://github.com/Daveiano/weewx-wdc/wiki/Configuration#persisting-changes-to-the-skinconf-between-updates

If this helps in any way?

But there is currently no way in preserving changes made to the templates and I don't think there is one way to implement this - but I am open for ideas.

**Reply by JonFear** [reply]
Got it, seems simple enough, I will do my updates there and check it works as we progress through future evolutions. Need to think about any additional pages per our other discussion however, same principle would apply I guess...
